### PR TITLE
adding yml files to add code coverage reports to .net pipeline

### PR DESCRIPTION
**Pull request by seankane-msft**
Adding code coverage report to the pipelines

### CODE REVIEW

**Comment by heaths** [comment]
Thank you for your contribution. I'm actually about to publish a PR that not only adds code coverage to our Azure Pipelines, but develop builds as well. We'll be expanding on that to cover more scenarios. I can add you to the PR I'll be creating very soon if you'd like.

File: eng/pipelines/templates/jobs/archetype-sdk-tests-jobs.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@ parameters:
 - name: UsePlatformContainer
   type: boolean
   default: false
+- name: RunCoverage
+  type: boolean
+  default: ne(varaibles['CoverageArg'], '--disablecov')
```
Comment (heaths) [inline eng/pipelines/templates/jobs/archetype-sdk-tests-jobs.yml:40]: `variables`

File: eng/pipelines/templates/steps/publish-coverage.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+parameters:
+  RunCoverage: false
+
+steps:
+
+  - task: PublishCodeCoverageResults@1
```
Comment (heaths) [inline eng/pipelines/templates/steps/publish-coverage.yml:6]: What's generating the actual report file? We disabled coverlet currently but I'm about to re-enable it.

**Review by heaths — CHANGES_REQUESTED** [review]
Thank you for your contribution. I'm actually about to publish a PR that not only adds code coverage to our Azure Pipelines, but develop builds as well. We'll be expanding on that to cover more scenarios. I can add you to the PR I'll be creating very soon if you'd like.

**Comment by heaths** [comment]
See #17067

**Comment by jsquire** [comment]
Hi @seankane-msft.  There hasn't been recent engagement on this PR and it looks to have been superseded by Heath's changes .  Would you please be so kind as to let us know if this is still an active work stream or if the PR should be closed out?

**Comment by seankane-msft** [comment]
@jsquire My apologies, I intended to close this PR.

**Comment by jsquire** [comment]
> @jsquire My apologies, I intended to close this PR.

Thanks for the quick response, and no need for apologies;  we're just working to keep things from getting stale.  😄

**Pull request closed without merging**